indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -1,13 +1,17 @@
 <h6>{{ work.title }}</h6>
 <p class='text-muted mb-1'>{{ work.frbr_uri }}</p>
 
-<p class='mb-1'>Published
-  {% if publication_name %} in {{ work.publication_name }} {% endif %}
-  on {{ work.publication_date|date:"Y-m-d" }}
-</p>
 {% if work.assent_date %}
   <p class='mb-1'>Assented to on {{ work.assent_date|date:"Y-m-d" }}</p>
 {% endif %}
+<p class='mb-1'>Published
+  on {{ work.publication_date|date:"Y-m-d" }}
+  {% if work.publication_name %}<br>in {{ work.publication_name }}{% endif %}
+  {% if work.publication_number %}{{ work.publication_number }}{% endif %}
+  {% if work.publication_document %}
+    <br><i class="fas fa-file text-muted"></i> <a href="{% url 'work_publication_document' frbr_uri=work.frbr_uri filename=work.publication_document.filename %}" target="_blank" rel="noopener">{{ work.publication_document.filename }}</a>
```

Comment:
This will be super verbose and redundant. Instead, I think the text above should be the link.

`Published on 1980-01-01 in _Western Cape Gazette no. 1234_`

Reply:
Also, pls make the `a` tag have `target="_blank" rel="noopener"`.